docs/user-documentation/CLAWfor1x.md

```diff
@@ -32,10 +32,10 @@ In Fedora 4 , what we would have called `objects` are now referred to as `resour
 * `Resource`: Roughly equivalent to a Fedora 3 object - a conceptual representation of a thing that can contain files or other containers.
 * `Non-RDF Source`: Roughly equivalent to a datastream. A Non-RDF Source (or binary) is simply a bitstream (e.g. JPG, PDF, XML, MP3, etc.).
 
-###Datastreams
-In Islandora CLAW, RDF datastreams (RELS-EXT and RELS-INT) are stored as RDF in Fedora. Binary datastreams (files, images) are files or `nonRdfResources` (see [PCDM](https://github.com/duraspace/pcdm/wiki)). Descriptive metadata datastreams (MODS, DC, DwC, PBCore, etc) are will be stored as RDF.
+### Datastreams
```

Comment:
If we are going to stick a term like `RDFSource` in here, we should probably link it to a definition. 
Like https://www.w3.org/TR/ldp/#dfn-linked-data-platform-rdf-source or even in https://www.w3.org/TR/rdf11-concepts/#h3_change-over-time ?